cpp/src/decisiontree/batched-levelalgo/quantiles.cu

```diff
@@ -0,0 +1,61 @@
+
+/*
+ * Copyright (c) 2019-2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <batched-levelalgo/quantiles.cuh>
+
+namespace ML {
+namespace DT {
+
+template <typename T>
+__global__ void computeQuantilesKernel(
```

Comment:
Am I seeing things or is this kernel defined twice?

Reply:
oh that's on me 😅 thank you for the catch!